### PR TITLE
Update screenshots with new space cards

### PR DESCRIPTION
**Pull request by misterpantz**
#### Description

- Update screenshots with new space cards
- Replace a lot of spaces page screenshots with space overview/dash screenshots
- Add rate limit info to gh app auth

#### Related issue

none

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [ ] New content
- [x] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [ ] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

**Review by SEAjamieD — APPROVED** [review]
🎉

File: static/img/platform/security/spaces.png
Comment (scottopherson) [inline static/img/platform/security/spaces.png:1]: These descriptions have the "No description provided for this space" styling (darker gray, centered-aligned). These descriptions should have lighter gray color and left-aligned text.

Comment (misterpantz) [reply static/img/platform/security/spaces.png:1]: Ohhh, thanks @scottopherson! I made them in my image editor, obviously. 😆  I'll fix.

Comment (misterpantz) [reply static/img/platform/security/spaces.png:1]: I was kind of wondering about the text being indented like that.

**Pull request merged**